apps/web/src/containers/LandingPageView/components/hero/components/hero-content.tsx

```diff
@@ -2,19 +2,15 @@ import { styled, Box, useTheme } from '@mui/material';
 import { Body, H1, Button } from '@notional-finance/mui';
 import { colors } from '@notional-finance/styles';
 import { Network } from '@notional-finance/util';
+import { useConnectWallet } from '@web3-onboard/react';
```

Comment:
Can you wrap this useConnectWallet into another hook so we can replace it with wagmi easier?